inst/app/app.R

```diff
@@ -0,0 +1,237 @@
+library(igvShiny)
```

Comment:
please use the currently available demo (see previous comment).

Reply:
ok